#!/usr/bin/env python
PACKAGE = "moverobotic_simple_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *
import math


gen = ParameterGenerator()

gen.add("base_frame_id", str_t, 0, "Name frame id for base", "base_link")
gen.add("map_frame_id", str_t, 0, "Name frame id for map", "map")
gen.add("home_coordinate", str_t, 0, "The home coordinate and direction syntax x y z yaw pitch roll", "0 0 0 0 0 0 1")
gen.add("home_get", bool_t, 0, "Click to get new home coordinate", False)
gen.add("home_set", bool_t, 0, "Click to set new home coordinate", False)
gen.add("home_button", int_t, 0, "Select home button", 0, 0, 8)

exit(gen.generate(PACKAGE, "moverobotic_simple_navigation", "moverobotic_simple_navigation")) #package_name, node_name, prefix_name_header_file (<prefix_name_header_file>Config.h)
